examples/vercel-kv-example/.env.example

```
@@ -0,0 +1,4 @@
+# Vercel KV credentials
+# Get these from your Vercel dashboard → Storage → your KV store → Settings
+KV_REST_API_URL=https://your-project.kv.vercel-storage.com
+KV_REST_API_TOKEN=your_api_token_here
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The KV_REST_API_TOKEN key should go before the KV_REST_API_URL key

(UnorderedKey)